test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Date.Tests.ps1

```diff
@@ -37,6 +37,14 @@ Describe "Get-Date DRT Unit Tests" -Tags "CI" {
         Get-date -Date 1/1/0030 -uformat %S%T%u%U%V%w%W%x%X%y%Y%% | Should be "0000:00:002012001/01/3000:00:00300030%"
     }
 
+    It "Passing <name> to -uformat produces a descriptive error" -TestCases @(
+        @{ name = "null"; value = $null }
```

Comment:
Please change `"null"` to `"$null"` (use backtick to escape the dollar sign).  Also add some whitespace to align `value` with test case below to make it easier to read:

```powershell
@{ name = "`$null"      ; value = $null }
@{ name = "empty string"; value = "" }
```

Reply:
@SteveL-MSFT Done. Thank you for the feedback -- it will be helpful moving forward.